doc/modules/mod_mam.md

```diff
@@ -209,6 +209,21 @@ These options will only have effect when the `rdbms` backend is used:
 
 Enables Archive ID to integer mappings cache.
 
+If caching is enabled, by default it will spawn its own [segmented cache](https://github.com/esl/segmented_cache) cache, with defaults as in [`mod_cache_users`](./mod_cache_users.md). To change these defaults, the same config can be accessed within the `cache_config` key. To see details about the meaning of each flag, see [`mod_cache_users`](./mod_cache_users.md). To reuse the cache already created by `mod_cache_users`, see the option below.
+
+```toml
+modules.mod_mam_meta.cache_config.strategy
+modules.mod_mam_meta.cache_config.time_to_live
+modules.mod_mam_meta.cache_config.number_of_segments
+```
+
+#### `modules.mod_mam_meta.cache_config.cache_module`
```

Comment:
We removed all the `config` words from the config file as the whole file is "config". Please just call it `cache.module`. Simplicity wins here IMO.